modmesh/app/euler1d.py

```diff
@@ -493,24 +432,21 @@ def set_solver_config(self):
 
         :return None
         """
-        self.init_solver(gamma=self.solver_config.get_var("gamma", "value"),
-                         pressure_left=self.solver_config.get_var("p_left",
-                                                                  "value"),
-                         density_left=self.solver_config.get_var("rho_left",
-                                                                 "value"),
-                         pressure_right=self.solver_config.get_var("p_right",
-                                                                   "value"),
-                         density_right=self.solver_config.get_var("rho_right",
-                                                                  "value"),
-                         xmin=self.solver_config.get_var("xmin", "value"),
-                         xmax=self.solver_config.get_var("xmax", "value"),
-                         ncoord=self.solver_config.get_var("ncoord", "value"),
-                         time_increment=(self.solver_config.
-                                         get_var("time_increment", "value")))
+        self.init_solver(gamma=self.solver_config["gamma"]["value"],
+                         pressure_left=self.solver_config["p_left"]["value"],
+                         density_left=self.solver_config["rho_left"]["value"],
+                         pressure_right=self.solver_config["p_right"]["value"],
+                         density_right=self.solver_config["rho_right"]
+                                                         ["value"],
+                         xmin=self.solver_config["xmin"]["value"],
+                         xmax=self.solver_config["xmax"]["value"],
+                         ncoord=self.solver_config["ncoord"]["value"],
+                         time_increment=(self.solver_config["time_increment"][
+                                                            "value"]))
         self.current_step = 0
-        self.interval = self.solver_config.get_var("timer_interval", "value")
-        self.max_steps = self.solver_config.get_var("max_steps", "value")
-        self.profiling = self.solver_config.get_var("profiling", "value")
+        self.interval = self.solver_config["timer_interval"]["value"]
```

Comment:
The original `get_var` handles the case if the key doesn't exist.
Perhaps write it as  `self.solver_config.get("timer_interval", {}). get("value", None)`

Reply:
But the `get_var` way was already quite wordy, and the new way seems to make it even wordy.

Reply:
API conciseness is hard, and that is why I developed the toggle code:
 https://github.com/solvcon/modmesh/blob/194a313d8bd6c898d959841d8288f69ac85c0692/tests/test_toggle.py#L258

It may take some efforts to connect the toggle library with PUI, so I am not promoting it now.  But we will need it at some point.